src/images.rs

```diff
@@ -1,6 +1,7 @@
 pub mod coblox_bitcoincore;
 pub mod dynamodb_local;
 pub mod elasticmq;
+pub mod eventstoredb;
```

Comment:
We usually name these modules after the Docker image name which in this case is `eventstore`.